app/Http/Controllers/DashboardController.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Exception;
+use Auth;
+use App\Models\User;
+use App\Models\Webhook;
+use App\Models\Bot;
+use App\Models\PayloadHistory;
+use App\Models\MessageHistory;
+use Illuminate\Http\Request;
+use DateTime;
+use DatePeriod;
+use DateInterval;
+use App\Enums\PayloadHistoryStatus;
+use App\Enums\MessageHistoryStatus;
+
+class DashboardController extends Controller
+{
+    public function index(Request $request)
+    {
+        $statisticParams = $this->handleStatisticParam($request->get('statistic'));
+        
+        $countData = [
+            'user' => User::count(),
+            'enabledWebhook' => Webhook::enable()->count(),
+            'disabledWebhook' => Webhook::disable()->count(),
+            'bot' => Bot::count(),
+        ];
+
+        $period = $this->getDateFromRange($statisticParams['fromDate'], $statisticParams['toDate']);
+        $payloadFailded = PayloadHistory::dataChart($statisticParams, PayloadHistoryStatus::FAILED);
+        $payloadSuccess = PayloadHistory::dataChart($statisticParams, PayloadHistoryStatus::SUCCESS);
+        $messageFailed = MessageHistory::dataChart($statisticParams, MessageHistoryStatus::FAILED);
+        $messageSuccess = MessageHistory::dataChart($statisticParams, MessageHistoryStatus::SUCCESS);
+
+        $payloadFaildedQuantity = $this->getDataChart($payloadFailded, $period);
+        $payloadSuccessQuantity = $this->getDataChart($payloadSuccess, $period);
+        $messageFaildedQuantity = $this->getDataChart($messageFailed, $period);
+        $messageSuccessQuantity = $this->getDataChart($messageSuccess, $period);
+
+        $payloadFailedChart =  $this->convertArrayChart($payloadFaildedQuantity, $period);
+        $payloadSuccessChart = $this->convertArrayChart($payloadSuccessQuantity, $period);
+        $messageFailedChart = $this->convertArrayChart($messageFaildedQuantity, $period);
+        $messageSuccessChart = $this->convertArrayChart($messageSuccessQuantity, $period);
+        $dateChart = $this->convertArrayChart($period);
+
+        $payloadChartData = [
+            'failedCases' => array_sum($payloadFaildedQuantity),
+            'successCases' => array_sum($payloadSuccessQuantity),
+            'dateChart' => $dateChart,
+            'payloadFailedChart' => $payloadFailedChart,
+            'payloadSuccessChart' => $payloadSuccessChart
+        ];
+        $messageChartData = [
+            'failedCases' => array_sum($messageFaildedQuantity),
+            'successCases' => array_sum($messageSuccessQuantity),
+            'dateChart' => $dateChart,
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/DashboardController.php</br>**Line**: 61</br>Each line in an array declaration must end in a comma